workflow/rules/detected_variants.smk

```diff
@@ -0,0 +1,36 @@
+__author__ = "Massimiliano Volpe"
+__copyright__ = "Copyright 2022, Massimiliano Volpe"
+__email__ = "massimiliano.volpe@liu.se"
+__license__ = "GPL-3"
+
+
+rule summary_detected_variants:
+    input:
+        vcf="pgx/annotation_variant_annotator/{sample}_{type}_{chr}.output.vcf",
+    output:
+        csv="pgx/summary_detected_variants/{sample}_{type}_{chr}.output.csv",
+    params:
+        target_bed=config.get("summary_detected_variants", {}).get("target_rsid", ""),
+    #       hidden_haplotypes = config.get("detected_variants", {}).get("hidden_haplotypes", "")
```

Comment:
```suggestion
```

Reply:
Yeah, I think this is not needed since the original script is not passing this file to the function. I also had a problem converting the original script, I fixed it in a way but I don't know, maybe it is good if you have the time to check it. My troubleshooting is reported [here](https://github.com/genomic-medicine-sweden/pgx/issues/28). The issue for me arises at [Line 86](https://github.com/JoelAAs/pgx_module/blob/master/src/Summary/get_target_variants.py) of the original script and as I understand, it tries to extract an index, possibly an integer number but gets an object instead. It serves just to get the FORMAT string from the VCF file in order to split it by colon ":" to get the values ['GT', 'AD', 'AF', 'DP', 'GQ', 'PL'] and naming correctly the different column in the resulting CSV file. I believed it was safe to assume that if the input VCF is not empty (the function checks for this) then you can extract this string from the first row.